pkg/providers/v1/tags.go

```diff
@@ -310,3 +310,48 @@ func (t *awsTagging) buildTags(lifecycle ResourceLifecycle, additionalTags map[s
 func (t *awsTagging) clusterID() string {
 	return t.ClusterID
 }
+
+func (c *Cloud) TagResource(resourceId string, tags map[string]string) error {
+	request := &ec2.CreateTagsInput{
+		Resources: []*string{aws.String(resourceId)},
```

Comment:
If this takes in a list then should the function also take in a list of `resourceIds`?

Reply:
Same for https://github.com/kubernetes/cloud-provider-aws/pull/308#discussion_r833968442

Reply:
resolved offline